### PR TITLE
Updating greenscale script

### PR DESCRIPTION
**Pull request by akanshaVashisth**
Hi team,

I have made changes in the script: color.py(old name), now called greenscale.py. 
Kindly review it and provide the code review comments.

Thanks!

### CODE REVIEW

**Comment by jackattackyang** [comment]
https://github.com/UBC-MDS/filterizePy/blob/493d45456669f9d8d0daecda417d2233ae17f1df/filterizePy/greenscale.py#L1-L2

Should we move these functions to __init__ since they are common to all of our functions

**Comment by akanshaVashisth** [comment]
Thanks for the review @chaomander2018  . I will add an example in the docstring. 
Happy Valentine's day to you too.

**Comment by jackattackyang** [comment]
> [filterizePy/filterizePy/greenscale.py](https://github.com/UBC-MDS/filterizePy/blob/493d45456669f9d8d0daecda417d2233ae17f1df/filterizePy/greenscale.py#L1-L2)
> 
> Lines 1 to 2 in [493d454](/UBC-MDS/filterizePy/commit/493d45456669f9d8d0daecda417d2233ae17f1df)
> 
>  import numpy as np 
>  import skimage.io 
> Should we move these functions to **init** since they are common to all of our functions

On second thought, these functions would still have to be imported from __init__, we are just saving one line of code. Leaving it is good!

File: filterizePy/test/test_greenscale.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    assert np.array_equal(output, gs_test_img), "The greenscale function does not work properly."
+
+
+def test_greenscale_black():
+    # Black image testing
+    test_img_black = np.array([[[0, 0, 0]]], dtype = "uint8")
+    gs_test_img_black = np.array([[[0, 0, 0]]], dtype = "uint8")
+
+    greenscale("../../img/test_img_black.jpeg")
+    output = skimage.io.imread("../../img/test_img_black.jpeg")
```
Comment (chaomander2018) [inline filterizePy/test/test_greenscale.py]: I would name it `expected_output` or something so to not be confused with the output from your green scale function

Comment (chaomander2018) [reply filterizePy/test/test_greenscale.py]: Everything else looks good to me! Great work!

Comment (akanshaVashisth) [reply filterizePy/test/test_greenscale.py]: Sure that makes it more clear. Thanks!

File: filterizePy/test/test_greenscale.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    greenscale(input_img)
+    input_img = skimage.io.imread('../../img/test_original.jpg')
+    output_img = skimage.io.imread('../../img/gs_test_original.jpg')
+    assert imghdr.what(output_img) in ['png','jpeg','gif','bmp','jpg'] and imghdr.what(output_img)==imghdr.what(input_img),"The output image has a different file format"
+
+def test_input_img():
+    input_img = "../../img/test_original.jpg"
+    greenscale(input_img)
+    input_img = skimage.io.imread('../../img/test_original.jpg')
+    output_img = skimage.io.imread('../../img/gs_test_original.jpg')
```
Comment (jackattackyang) [inline filterizePy/test/test_greenscale.py]: For exception raising here, we don't need to pass in the `input_img` and `output_img`. The below should suffice:
```
with pytest.raises(FileNotFoundError):
        greenscale("not a file path")
```

Comment (akanshaVashisth) [reply filterizePy/test/test_greenscale.py]: Yes, that's true. Thanks for catching that!

**Comment by jackattackyang** [comment]
Looks good!

**Pull request merged**